ev-dev-tools/src/ev_cli/templates/interface-Base.hpp.j2

```diff
@@ -8,12 +8,39 @@
 #include <framework/ModuleAdapter.hpp>
 #include <utils/types.hpp>
 #include <utils/error.hpp>
+#include <utils/error/error_state_monitor.hpp>
+#include <utils/error/error_manager_impl.hpp>
+#include <utils/error/error_factory.hpp>
 
 #include "Types.hpp"
 
 class {{ info.class_name }} : public Everest::ImplementationBase {
 public:
-    {{ info.class_name }}(Everest::ModuleAdapter* ev, const std::string& name) : Everest::ImplementationBase(), _ev(ev), _name(name) {};
+    {{ info.class_name }}(Everest::ModuleAdapter* ev, const std::string& name)
+        : Everest::ImplementationBase(),
+        _ev(ev),
+        _name(name) {
+        if (ev == nullptr) {
```

Comment:
are these nullptr checks really needed here, shouldn't everest-framework always provide us with non-null parameters for this?

Reply:
I added it, because there are unittests in EnergyManager and EvseManager that use "Stub" classes that inherit from class `ModuleAdapter` and also classes that inherit from the auto-generated "Impl"-classes. Those are initialized sometimes with nullptr in module_adapter attribute... This is why I wanted to add log in those cases

In general this testing approach is something that is a bit tricky to do with new `ErrorManager` classes, since they don't have an abstract class they inherit from which then could be implemented in another way for testing purposes..
I planned to do there some improvements in terms of testability

Reply:
ok seems reasonable for now